docs/compatibility-table.mdx

```diff
@@ -11,14 +11,16 @@ is by always using the latest version. However,
 if for some reason that isn't possible, you can refer to 
 the table below to assess which version you should use.
 
-|   patrol_cli   |     patrol     |
-| -------------- | -------------- |
-| 3.2.0 -        | 3.11.0 -       |
-| 3.1.0 - 3.1.1  |     3.10.0     |
-| 2.6.5 - 3.0.1  | 3.6.0 - 3.10.0 |
-| 2.6.0 - 2.6.4  | 3.4.0 - 3.5.2  |
-| 2.3.0 - 2.5.0  | 3.0.0 - 3.3.0  |
-| 2.2.0 - 2.2.2  | 2.3.0 - 2.3.2  |
-| 2.0.1 - 2.1.5  | 2.0.1 - 2.2.5  |
-| 2.0.0          | 2.0.0          |
-| 1.1.4 - 1.1.11 | 1.0.9 - 1.1.11 |
+| patrol_cli            | patrol            | Flutter Version |
+|-----------------------|-------------------|-----------------|
+| 3.2.0+                | 3.11.0+           | 3.22+           |
+| 3.1.0 - 3.1.1         | 3.10.0            | 3.10 – 3.15     |
```

Comment:
Now there is a gap between `3.15` and `3.22`. I think it would be a better idea to name this column `Min Flutter Version` and fill it with minimum supported version of Flutter instead of a range